### PR TITLE
Deduplicate common flags

### LINKED ISSUE #271: deduplicate common flags
does it make sense to do this? seems like it would help with standardization (of helpers) & less technical debt.

### PR DESCRIPTION
**Pull request by zramsay**
- contracts was left alone because wonkyness with epm,
- flags that did not have a duplicate were not deduplicated into `flags.go` (but maybe they should, then all flags are in one place...?) and
- `func buildFlag()` in `flags.go` is ~messy and can probably be refactored to be clearer about which commands use which flags.

this PR closes #271

### CODE REVIEW

File: commands/old_falgs.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
+package commands
```
Comment (compleatang) [inline commands/old_falgs.go]: is this file doing anything?

File: commands/contracts.go [hunk 2]
```diff
@@ -78,7 +80,7 @@ deploy types.
 }
 
 //----------------------------------------------------
-
+// XXX todo deduplicate flags -> [zr] things get wonky with epm
```
Comment (compleatang) [inline commands/contracts.go]: i can work through this tomorrow.

File: commands/old_falgs.go (hunk 1 above)
Comment (zramsay) [inline commands/old_falgs.go]: no. it's merely sentimental / easier to revert/debug when something breaks

File: commands/contracts.go (hunk 2 above)
Comment (zramsay) [inline commands/contracts.go]: most likely has to do with the wrapping

**Pull request merged**